.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: build
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Downcase repo
+        run: echo REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
+      -
+        name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: ghcr.io/${{ env.REPOSITORY }}
+          tag-sha: true
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to GitHub Container Registry
```

Comment:
I don't think deploying to GitHub registry is the best solution.
I think it will be more user friendly to deploy it to DockerHub.

Reply:
Well, given the [upcoming changes](https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/) in DockerHub's pull limits, a lot of projects are moving to GitHub registry (including LinuxServer.io). From my personal point of view, it's also a bit easier to manage.

And from a user's point of view, the only difference is `docker pull ghcr.io/noahcardoza/cloudproxy` instead of `docker pull noahcardoza/cloudproxy`, not a huge pain IMO...

That being said, this PR proposes one way of doing it. Of course, that'll be up to @NoahCardoza to make the call as the project owner :)

Reply:
I wonder if it would be possible to push to both? Then it's up to the discretion of the user and what they prefer.

Reply:
It is of course possible, see the docs [here](https://github.com/marketplace/actions/build-and-push-docker-images#push-to-multi-registries) :)

Reply:
As soon as @Twanislas merges https://github.com/Twanislas/CloudProxy/pull/1 both options will be available.

@ngosang check out:
- https://hub.docker.com/r/noahcardoza/cloudproxy/tags/container/package/cloudproxy
- https://github.com/users/NoahCardoza/packages

I'll be clearing all those images out before I merge this branch since those were for testing.